libs/ledger-live-common/src/operation.ts

```diff
@@ -178,7 +178,6 @@ export function getOperationAmountNumber(op: Operation): BigNumber {
     case "REWARD":
     case "REWARD_PAYOUT":
     case "SUPPLY":
-    case "WITHDRAW":
```

Comment:
This is currently used by Celo integration, are you sure it won't break something?

Reply:
@haammar-ledger I've tested and nothing looks to be broken - it will now just show a negated fee next to withdraw operations instead of a positive value. Which as I understand is the expected behavior, as the staked/locked funds should not look like they leave your account in the first place.

However if it is a best practice to use a different set of operation types for each integration I'm happy to rename it to something like `"WITHDRAW_UNSTAKED"`, let me know.

Reply:
I believe the idea for Celo was to display the amount that was actually withdrawn to the user's account, hence positive, and represented by `op.value`.
Do you confirm this isn't the desired behavior for Near? If so then indeed, a dedicated operation type can be created.

Reply:
@haammar-ledger looked into it further - this makes sense. Celo shows negative for locking, makes sense that you'd get a positive for unlocking.

On NEAR it's different - we don't have access to the `value` for withdrawing, so we just negate the fee for each of the staking operations (staking, unstaking, withdrawing). Because this is the case, I've added a new type in this commit: https://github.com/LedgerHQ/ledger-live/pull/1126/commits/887c88a1e4ed190bca9bc4fec7ea7b6dd503a95c

So ready for another look!

Reply:
LGTM! :+1: